ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/ImageOverviewTable.tsx

```diff
@@ -215,6 +217,29 @@ function ImageOverviewTable({
                         const watchImageMenuText = isWatchedImage ? 'Unwatch image' : 'Watch image';
                         const watchImageMenuAction = isWatchedImage ? onUnwatchImage : onWatchImage;
 
+                        const rowActions: IAction[] = [];
+
+                        if (hasWriteAccessForWatchedImage && name?.tag) {
+                            rowActions.push({
+                                title: watchImageMenuText,
+                                onClick: () =>
+                                    watchImageMenuAction(
+                                        `${name.registry}/${name.remote}:${name.tag}`
+                                    ),
+                            });
+                        }
+
+                        if (hasGenerateSBOMAbility) {
+                            rowActions.push({
+                                title: 'Generate SBOM',
+                                isAriaDisabled: !isScannerV4Enabled,
+                                description: !isScannerV4Enabled
+                                    ? 'SBOM generation requires Scanner V4 to be enabled'
```

Comment:
An observation, not necessarily a change request, that text has operator precedence question:
* SBOM generation / requires / Scanner V4 to be enabled
* (For) SBOM generation (to be enabled) / requires / Scanner V4
* In this case, ambiguity might be neutral, because **to be enabled** could apply to both

Reply:
Oh good point. How about:

"Scanner V4 must be enabled to generated SBOM files"